client/src/components/graph/graph.tsx

```diff
@@ -1083,6 +1211,7 @@ const StillLoading = ({ displayName, width, height }: any) => (
         alignItems: "center",
       }}
     >
+      <Button minimal loading intent="primary" />
       <span style={{ fontStyle: "italic" }}>Loading {displayName}</span>
     </div>
   </div>
```

Comment:
Can you include a screen cap of this new button? I'm not sure what the purpose is

Reply:
This is the spinning loader that appears when the embedding is loaded

Reply:
<img width="460" alt="Screenshot 2024-03-06 at 8 19 17 PM" src="https://github.com/chanzuckerberg/single-cell-explorer/assets/7976579/933e2266-95dc-47ce-a96e-519d3ab17b3e">